.storybook/__mocks__/react-native-gesture-handler.tsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
```

Comment:
Do we see a situation where we'd add more mocks in the future?  If so, should this file also be called a version of `mocks`  and we keep adding to it, or do you prefer to have a file for each?

Reply:
yup. was just to lazy to copy the one in mobile. leaving it as is until that day comes